npc/re/quests/quests_17_1.txt

```diff
@@ -0,0 +1,8942 @@
+		mes "It seems that power plant supports the entire mansion.";
+		next;
+		mes "[Rookie]";
+		mes "Through the power plant, you can get to the garden of mansion.";
+		mes "The mansion's manager is staying near the garden. be sure to meet them first.";
+		next;
+		select("Manager?");
+		mes "[Rookie]";
+		mes "If you look around closely, you'll understand.";
+		mes "They seem to be having a hard time because of the intruders.";
+		next;
+		mes "[Rookie]";
+		mes "Many other volunteer adventurer have entered before us.";
+		mes "The sewage plant is quite complicated and you could get lost.";
+		next;
+		mes "[Rookie]";
+		mes "There might be a colleague that ^4D4DFFfell behind^000000, Please move together with them.";
+		mes "Before you register as a guest in the mansion, prioritize everyone's safety first.";
+		next;
+		mes "[Rookie]";
+		mes "Then, this way.";
+		completequest 18000;
+		setquest 18001;
+		close2;
+		cutin "",255;
+		warp "ba_pw02",275,20;
+		end;
+	}
+	if (isbegin_quest(18001) == 1) {
+		mes "[Rookie]";
+		mes "Go ahead and enter the mansion through this sewer.";
+		mes "Most of the colleagues who went in first cleaned up already.";
+		next;
+		mes "[Rookie]";
+		mes "There might be a colleague that ^4D4DFFfell behind^000000, Please move together with them.";
+		mes "Before you register as a guest in the mansion, prioritize everyone's safety first.";
+		close;
+	}
+	mes "[Rookie]";
+	mes "The path we took with you adventurer is a fresh feeling for me.";
+	mes "I feel really alive these days.";
+	close;
+*/ // Episode 17.2 - END
+	end;
+
+OnInit:
+	questinfo (QTYPE_QUEST, QMARK_YELLOW, " isbegin_quest(16360) == 1 ");
+	end;
+}
+
+//= Finding Patrick Kaufmann
+sp_cor,132,144,3	script	Worker#171_rvs_11	4_F_05,{
+	if (isbegin_quest(17020) == 1) {
+		mes "[Worker]";
+		mes "I'll have to split the workforce because I only have so few people remaining.";
+		npctalk "I have to split the workforce.","",bc_self;
+		next;
+		mes "[Worker]";
+		mes "We'll have to take turns.";
+		npctalk "No, that's not very productive.","Worker#171_rvs_13",bc_self;
+		next;
+		select("Do you know Patrick Kaufmann?");
+		mes "[Worker]";
+		mes "Who?";
+		npctalk "Pat, who?","Worker#171_rvs_12",bc_self;
+		next;
+		select("Patrick Kaufmann. Facility Maintenance Officer.");
+		mes "[Worker]";
+		mes "Patrick Kaufmann? Don't you mean Patrick Young?";
+		npctalk "I don't think that's the right name.","Worker#171_rvs_14",bc_self;
+		next;
+		mes "[Worker]";
+		mes "He's the logistic guy.";
+		npctalk "Facility Maintenance Department....","Worker#171_rvs_13",bc_self;
+		next;
+		mes "[Worker]";
+		mes "Where's the facility department here?";
+		npctalk "Don't you know?","",bc_self;
+		next;
+		mes "[Worker]";
+		mes "I knew before I came here!";
+		npctalk "That's not possible.","Worker#171_rvs_14",bc_self;
+		next;
+		mes "[Worker]";
+		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual
```

Comment:
```suggestion
		mes "That's right, that's right! The guy who looked tall and big. I wanted to be his friend because we came from the same place. I couldn't see him one day. Then I was moved.";
```